packages/rest/src/keys.ts

```diff
@@ -3,37 +3,30 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {CoreBindings} from '@loopback/core';
 import {BindingKey, Context} from '@loopback/context';
-
-/**
- * See https://github.com/Microsoft/TypeScript/issues/26985
- */
-// import {OpenApiSpec} from '@loopback/openapi-v3-types';
-import {OpenAPIObject as OpenApiSpec} from 'openapi3-ts';
```

Comment:
https://github.com/Microsoft/TypeScript/issues/26985 has been fixed, the workaround is no longer needed.